common/api/summary/appui-abstract.exports.csv

```diff
@@ -2,54 +2,20 @@ sep=;
 Release Tag;API Item
 public;AbstractActionItemProps 
 public;AbstractMenuItemProps 
-public;AbstractStatusBarActionItem 
-deprecated;AbstractStatusBarActionItem 
-public;AbstractStatusBarCustomItem 
-deprecated;AbstractStatusBarCustomItem 
-public;AbstractStatusBarItem 
-deprecated;AbstractStatusBarItem 
-public;AbstractStatusBarItemUtilities
-deprecated;AbstractStatusBarItemUtilities
-public;AbstractStatusBarLabelItem 
-deprecated;AbstractStatusBarLabelItem 
 public;AbstractToolbarProps
-public;AbstractWidgetProps 
```

Comment:
We can remove it.
We'll have to fix this on `appui` side once 4.0 is out (since the type will no longer exist when building w/ 4.0).
That (already deprecated) helper exists for consumers using `appui` w/ 3.7 core versions only.

Reply:
It will be compatible.
The API will still be available for both 3.x and 4.x, but the types will be unknown in case of 4.0 and fallback to `any` by TS (from the consumer side). And should not be used if using 4.0.
But it's a good point, that we should've defined allowed versions better in the package.json of appui. Since we are saying we support >=3.x, even though 4.0 is not even out yet? How about 5.0...?

Reply:
> but the types will be unknown in case of 4.0 and fallback to any by TS
Ok, I didnt test, I expected this to simply not build.